packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -406,6 +411,11 @@ export class PeerDiscovery {
     }
   }
 
+  private isPeerCoolingDown(peer: CachedENR): boolean {
+    const peerScore = this.peerRpcScores.getScore(peer.peerId);
+    return peerScore < 0;
```

Comment:
`applyReconnectionCoolDown` we set `lastUpdate` in the future, so to detect cool down peer we need to check `lastUpdate`? need to move this method to `RealScore` in that case

Reply:
ok. updated